### PR TITLE
Enable writeback cache by default

### PR DESCRIPTION
**Pull request by mystenmark**
Writeback cache has been stable on all mysten fullnodes plus our testnet validator for several months. Time to enable it by default everywhere

### CODE REVIEW

**Comment by lxfind** [comment]
How does this enable the writeback cache by default exactly? Wouldn't we need to update the default value in the node config?

**Comment by halfprice** [comment]
Does everywhere mean fullnode everywhere, or sui node everywhere including validators?

**Comment by mystenmark** [comment]
> How does this enable the writeback cache by default exactly? Wouldn't we need to update the default value in the node config?

It enables it unless it is explicitly disabled in the config, but I don't think anybody will have disabled it via the config

**Comment by lxfind** [comment]
But in the node config, the default value is ExecutionCacheConfig::PassthroughCache, right?

**Comment by mystenmark** [comment]
> But in the node config, the default value is ExecutionCacheConfig::PassthroughCache, right?

Yes, you are right - just pushed a fix and verified that writeback cache is chosen at startup with a fresh config dir.

**Comment by mystenmark** [comment]
> Does everywhere mean fullnode everywhere, or sui node everywhere including validators?

All sui nodes that have not explicitly disabled it via config or env var

**Comment by lxfind** [comment]
Didn't see the fix?

**Comment by mystenmark** [comment]
> Didn't see the fix?

Sorry about that, thought I pushed it - now its there

File: crates/sui-config/src/node.rs [hunk 1]
```diff
@@ -206,14 +206,19 @@ pub struct NodeConfig {
...
-#[derive(Clone, Debug, Deserialize, Serialize, Default)]
+#[derive(Clone, Debug, Deserialize, Serialize)]
 #[serde(rename_all = "kebab-case")]
 pub enum ExecutionCacheConfig {
-    #[default]
     PassthroughCache,
-    WritebackCache {
-        max_cache_size: Option<usize>,
-    },
+    WritebackCache { max_cache_size: Option<usize> },
```
Comment (lxfind) [inline crates/sui-config/src/node.rs:213]: Could you also document the behavior when this is None? Not sure if it means unbounded cache or with some constant size

**Review by lxfind — APPROVED** [review]

**Review by halfprice — APPROVED** [review]

**Pull request merged**